api/models/Site.js

```diff
@@ -5,7 +5,11 @@
 * @docs        :: http://sailsjs.org/#!documentation/models
 */
 
-var DEFAULT_BUCKET = 'federalist.18f.gov.s3-website-us-east-1.amazonaws.com';
+var env = process.env.NODE_ENV;
+
+var DEFAULT_BUCKET = env === 'staging' ?
+  'federalist-staging.18f.gov.s3-website-us-east-1.amazonaws.com' :
+  'federalist.18f.gov.s3-website-us-east-1.amazonaws.com';
```

Comment:
I think we should go ahead and merge this, we'll still need some kind of mechanism for ensuring the user can get redirected to one of these URLs when the app is running locally (like manually setting env variables) and I don't want to have more moving parts to keep track of at this point